athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/glue/DefaultGlueType.java

```diff
@@ -71,10 +74,40 @@
         this.arrowType = arrowType;
     }
 
-    public static DefaultGlueType fromId(String id)
```

Comment:
Is this just the diff cutting weird or are we replacing all ids -> DefaultGlueType to just a string to decimal?

Reply:
Just the diff cutting weird, this function "fromId" still exists. See line 103 in the new version of the file on the right hand side.